server/src/main/java/org/solarbank/server/service/CalculateService.java

```diff
@@ -0,0 +1,35 @@
+package org.solarbank.server.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.solarbank.server.dto.UserInputDto;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CalculateService {
+    /**
+     * temporary method to return static mock data.
+     *
+     * @param userInputDto data from user input
+     * @return response returns calculated data
+     */
+    public Map<String, Object> processUserInput(UserInputDto userInputDto) {
+
+        Map<String, Double> energyGenPerMonth = new HashMap<>();
+        energyGenPerMonth.put("January", 0.1);
+        energyGenPerMonth.put("February", 0.2);
+
+        Map<String, Object> savingsPerYear = new HashMap<>();
+        savingsPerYear.put("currencyCode", "USD");
+        savingsPerYear.put("amount", 1000.0);
+
+        double energyGenPerYear = 1.0;
+
+        Map<String, Object> response = new HashMap<>();
+        response.put("energyGenPerYear", energyGenPerYear);
+        response.put("energyGenPerMonth", energyGenPerMonth);
+        response.put("savingsPerYear", savingsPerYear);
+
+        return response;
```

Comment:
In python and other non-strongly typed languages you often see objects containing mixed types. However, in Java we should avoid mixing types at all costs, and so returning `Object` is a code smell. In java `Object` represents the root parent of all classes and all classes extend it under the hood.

Instead of using a map in this case you should create a class which encapsulates the result of the calculation, where its fields are the different properties and can have different types.

Reply:
added CalculateResult class to encapsulate calculation result